combat-trainer.lic

```diff
@@ -2065,6 +2065,17 @@ class TrainerProcess
           fix_standing
         end
       end
+    when 'Ret Stealth'
+      retreat
+      if hide?(@hide_type) && !@dont_stalk && !game_state.npcs.empty? && @hide_type == 'hide'
```

Comment:
❓ Is this section a copy of the 'Stealth' block? If so, you should extract this to a re-used helper function.

Reply:
updated!  Thanks binu for some nudging.